### PR TITLE
Add files to generate Airflow's Python SDK

### PR DESCRIPTION
**Pull request by msumit**
Adding `python.sh` file to generate Python Client for Airflow using OpenApi3 specifications.

### CODE REVIEW

File: clients/gen/go.sh [hunk 1]
```diff
@@ -35,48 +30,11 @@ go_config=(
...
+validate_input "$@"
 
 gen_client go \
     --package-name airflow \
     --git-repo-id airflow-client-go/airflow \
     --additional-properties "${go_config[*]}"
 
-# patch generated client to support problem HTTP API
-# this patch can be removed after following upstream patch gets merged:
-# https://github.com/OpenAPITools/openapi-generator/pull/6793
```
Comment (msumit) [inline clients/gen/go.sh:65]: @houqp I see that this patch has been merged in openapi, so I think it should be safe to remove it now, though I'm getting some testcase failures and hoping that it's not related to this change.

Comment (houqp) [reply clients/gen/go.sh:65]: i think you would need to bump `OPENAPI_GENERATOR_CLI_VER` to the latest version to pick up the fix.

Comment (houqp) [reply clients/gen/go.sh:65]: also did you see test cases failing for go client or python client?

Comment (msumit) [reply clients/gen/go.sh:65]: Thanks @houqp, used the latest stable version of openapi gen cli. With that, I think we should bump up the go client version from 1.0.0 to 1.1.0? as I see some changes in the code structure/imports due to that.

Comment (houqp) [reply clients/gen/go.sh:65]: yep, we should bump client version for changes from code gen. whether it should be `1.1.0` or `1.0.1` depends on whether the new client as new feature or not.

**Comment by houqp** [comment]
Last time I tried it (with 4.x), it was still generating python client code with syntax errors in tests, is it still the case right now? If so, I can help dig into fixes for openapi generator.

File: clients/gen/go.sh (hunk 1 above)
Comment (msumit) [reply clients/gen/go.sh:65]: I believe there would be an addition of some more APIs, and a little bit of diff import structure as well, so we should go with `1.1.0` at least.

**Comment by msumit** [comment]
> Last time I tried it (with 4.x), it was still generating python client code with syntax errors in tests, is it still the case right now? If so, I can help dig into fixes for openapi generator.

@houqp there are no syntax errors, but the assertion errors. So I reverted the changes in v1.yaml for this PR but using them while generating the python client. [See this](https://github.com/apache/airflow/runs/2110985309?check_suite_focus=true)
Screenshot 2021-03-16 at 9 56 07 AM

File: clients/gen/go.sh (hunk 1 above)
Comment (houqp) [reply clients/gen/go.sh:65]: sounds good 👍

**Comment by msumit** [comment]
@houqp can you approve this PR?

**Comment by houqp** [comment]
@msumit sorry that i have been busy lately, would need some more time to test the change on my end. It looks good overall, but it's weird to see that content type assert failure prior to you reverting the change in openapi spec. It looks like a server error to me and I remember it used to behave correctly when I was testing the go client. In fact the go client code gen fix I sent upstream was for fixing this problem specifically.

**Review by houqp — APPROVED** [review]
Looks like some of the generated tests are still failing due to incorrect code gen:

```
test/test_task_collection.py:51: AttributeError
============================================================================================================== short test summary info ==============================================================================================================
FAILED test/test_dag.py::TestDAG::testDAG - NameError: name 'null' is not defined
FAILED test/test_dag_collection.py::TestDAGCollection::testDAGCollection - NameError: name 'null' is not defined
FAILED test/test_dag_detail.py::TestDAGDetail::testDAGDetail - NameError: name 'null' is not defined
FAILED test/test_dag_detail_all_of.py::TestDAGDetailAllOf::testDAGDetailAllOf - TypeError: __init__() got an unexpected keyword argument '__type'
FAILED test/test_dag_run_collection.py::TestDAGRunCollection::testDAGRunCollection - AttributeError: module 'airflow_client.models' has no attribute 'conf'
FAILED test/test_task.py::TestTask::testTask - AttributeError: module 'airflow_client.models.task_extra_links' has no attribute 'Task_extra_links'
FAILED test/test_task_collection.py::TestTaskCollection::testTaskCollection - AttributeError: module 'airflow_client.models.task_extra_links' has no attribute 'Task_extra_links'
=========================================================================================================== 7 failed, 111 passed in 1.01s ===========================================================================================================
```

But I don't think this should be blocking us from iterating on the client 👍  We just have get those issues fixed in openapi generator.

**Comment by msumit** [comment]
Thanks @houqp. Somehow all tests on the generated clients are successful for me.

`===========117 passed in 0.90s ============
`

Anyway, lets merge this for now and will look for the fixes in openapi.

**Pull request merged**